src/main/java/org/gridsuite/sensitivityanalysis/server/SensitivityAnalysisController.java

```diff
@@ -171,15 +186,15 @@ public ResponseEntity<Void> deleteResults() {
     @Operation(summary = "Get the sensitivity analysis status from the database")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The sensitivity analysis status")})
     public ResponseEntity<String> getStatus(@Parameter(description = "Result UUID") @PathVariable("resultUuid") UUID resultUuid) {
-        String result = service.getStatus(resultUuid);
-        return ResponseEntity.ok().body(result);
+        SensitivityAnalysisStatus result = service.getStatus(resultUuid);
+        return ResponseEntity.ok().body(result == null ? null : result.name());
```

Comment:
return ResponseEntity.ok().body(Optional.ofNullable(result).map(Result::name).orElse(null));

Reply:
I don't disagree but frankly I find this syntax heavy and not clearer. Is it recommended for the project? Is there a performance improvement or something?